pkgs/racket-doc/scribblings/reference/custom-write.scrbl

```diff
@@ -108,7 +108,7 @@ property, @racket[#f] otherwise.}
 
 
 @defproc[(custom-write-accessor [v custom-write?])
-         (custom-write? output-port? boolean? . -> . any)]{
+         (custom-write? output-port? (or/c boolean? (integer-in 0 1)) . -> . any)]{
```

Comment:
I'd rather `0 1` than `(integer-in ....)`. It's shorter, and then there's no need to think about what kind of interval `integer-in` gives.

I'd also prefer `#f #t` over `boolean?` but that's less likely to be confusing.

Reply:
`(or/c boolean? 0 1)` includes `0.0` and `1.0`, while `(or/c boolean? (integer-in 0 1))` requires `0` and `1` to be exact.

Reply:
I still prefer `0` and `1` for the docs, whether or not the implementation allows `0.0` and `1.1`

Reply:
I also tried `(one-of/c #t #f 0 1)`, which I expected to require exact `0` and `1` since the documentation says it uses `eqv?` as the comparison predicate. But that contract also let inexact `0.0` and `1.0` through. That might be a mistake in the implementation of `one-of/c`.

Reply:
Okay, I've changed it to use `(or/c #t #f 0 1)` for the documentation, even though it's less accurate.